src/elements/content-sidebar/__tests__/SidebarPanels.test.js

```diff
@@ -144,4 +149,21 @@ describe('elements/content-sidebar/SidebarPanels', () => {
             expect(instance.versionsSidebar.current.refresh).toHaveBeenCalledWith();
         });
     });
+
+    test('should reset the current version if the versions route is no longer active', () => {
+        const onVersionChange = jest.fn();
+        const wrapper = getWrapper({ location: { pathname: '/activity' }, onVersionChange });
+
+        wrapper.setProps({ location: { pathname: '/activity/versions/123' }, isOpen: true });
+        expect(onVersionChange).not.toBeCalled();
+
+        wrapper.setProps({ location: { pathname: '/activity/versions/123' }, isOpen: false });
+        expect(onVersionChange).not.toBeCalled();
+
+        wrapper.setProps({ location: { pathname: '/activity/versions/456' }, isOpen: true });
+        expect(onVersionChange).not.toBeCalled();
+
+        wrapper.setProps({ location: { pathname: '/details' } });
+        expect(onVersionChange).lastCalledWith(null);
```

Comment:
Would `.toBeCalledWith(null)` work here or is the number and order of calls important?

Reply:
It's from an earlier attempt to run a few assertions in series and I forgot to update it afterwards.
I just rearranged the tests and change it to `.toBeCalledWith(null)`.